lib/oli_web/live/common/properties/read_only.ex

```diff
@@ -23,6 +24,21 @@ defmodule OliWeb.Common.Properties.ReadOnly do
     """
   end
 
+  defp render_property(%{show_copy_btn: true} = assigns) do
+    copy_id = "copy-#{UUID.uuid4(:hex)}"
+
+    ~F"""
+    <div class="relative">
+      <input id={copy_id} class="form-control" type={@type} disabled value={@value}/>
```

Comment:
Adds Copy button to readonly URL field to making it easier to open this link

Reply:
Very nice addition!